src/main/kotlin/io/github/mojira/arisa/modules/CommandModule.kt

```diff
@@ -47,9 +49,11 @@ class CommandModule(
     }
 
     @Suppress("SpreadOperator")
-    private fun executeCommand(comment: String, issue: Issue, userIsMod: Boolean): Either<ModuleError, ModuleResponse> {
+    private fun executeCommand(comment: String, issue: Issue, userIsMod: Boolean, author: User):
+            Either<ModuleError, ModuleResponse> {
         val split = comment.split("\\s+".toRegex())
         val arguments = split.toTypedArray()
+        staffAuthor = author
```

Comment:
lets not use global variables like this to pass this info. 

1- If there are multiple commands you will grab the username of latest for all
2- Its bad practice

You can modify the return of the either to include the author username(no need forthe entire object, just get their name)